### PR TITLE
updated to es6 import

### PR DESCRIPTION
**Pull request by vwong125**
Added images for MJ tiles
Added ThreeJS cube to landing page

### CODE REVIEW

File: src/pages/Main.tsx [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
+import * as THREE from 'three';
 import { HistoryParams } from '../types/react-router';
 
+import CHUN from '../assets/tiles/Regular/Chun.png';
+import MAN1 from '../assets/tiles/Regular/Man1.png';
+import HATSU from '../assets/tiles/Regular/Hatsu.png';
+import NAN from '../assets/tiles/Regular/Nan.png';
+import PEI from '../assets/tiles/Regular/Pei.png';
+import SOU1 from '../assets/tiles/Regular/Sou1.png';
+
```
Comment (yummyblabla) [inline src/pages/Main.tsx:12]: This and all other Three related code. Move outside of page into a component in components/main. Pass the ref as a prop. Currently looks really messy.

Comment (yummyblabla) [reply src/pages/Main.tsx:12]: On second thought, you don't need to pass ref. Just do it in another component and it should work

File: src/components/MainThreeJSComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      new THREE.MeshBasicMaterial({ map: loader.load(PEI) }),
+      new THREE.MeshBasicMaterial({ map: loader.load(NAN) }),
+    ];
+
+    return materials;
+  };
+  const initThree = () => {
+    scene = new THREE.Scene();
+    const { clientHeight, clientWidth } = ThreeJSRefContainer.current;
+    camera = new THREE.PerspectiveCamera(75, clientWidth / clientHeight, 0.1, 1000);
```
Comment (jzc1828) [inline src/components/MainThreeJSComponent.tsx:43]: maybe consider putting these hardcoded values into constants?

File: src/components/MainThreeJSComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    renderer = new THREE.WebGLRenderer({ alpha: true });
+
+    renderer.setSize(clientWidth, clientHeight);
+    renderer.setClearColor(0x531cb3, 0.15);
```
Comment (jzc1828) [inline src/components/MainThreeJSComponent.tsx:47]: same here

**Pull request merged**